1-exercises/B-highlight-words/script.js

```diff
@@ -1,7 +1,30 @@
-function highlightWords(paragraph, colours) {
```

Comment:
your highlightWords function seem to be missing, let me know if you want a hand going through this